src/context/TransactionsProvider/TransactionsProvider.tsx

```diff
@@ -54,7 +54,7 @@ export const TransactionsProvider = ({ children }: TransactionsProviderProps): J
     // account specifiers changing will trigger this effect
     // we've disconnected/switched a wallet, unsubscribe from tx history and clear tx history
     if (!isSubscribed) return
-    moduleLogger.info('unsubscribing txs')
+    moduleLogger.debug('unsubscribing txs')
```

Comment:
Reduced the logging level here to avoid polluting the logs with not very helpful information.